app/assets/stylesheets/custom/layout.scss

```diff
@@ -54,6 +54,10 @@ footer {
 .top-bar {
   .top-bar-title {
     a img {
+      @include breakpoint(medium) {
```

Comment:
Hi @javierm, maybe now we can use the same rule for all the screens (`height: rem-calc(80);`). I tried it, and it seems to work mostly the same.

Reply:
I'm fine with all the options; using `width`, deleting the rule, or using `height` on all screens.

Up to you! I'm approving the pull request, and you can decide.